internal/repository/repository.go

```diff
@@ -515,6 +517,85 @@ func (r *Repository) LoadIndex(ctx context.Context) error {
 	return nil
 }
 
+const listPackParallelism = 10
+
+// LoadIndexFromPacks loads a new index by reading all given pack files
+// This will create one Index data structure per pack file read that is not
+// marked as finalized.
+func (r *Repository) LoadIndexFromPacks(ctx context.Context, packs map[restic.ID]int64, p *progress.Counter) (invalid restic.IDs, err error) {
```

Comment:
What about `ReindexPacks`?

Reply:
I agree that `Load` is not a good part of the name. But I also disliked `ReindexPacks` as this might indicate that the packs must be already part of the index. I changed the name to `CreateIndexFromPacks`.